website_logo/views/website_templates.xml

```diff
@@ -0,0 +1,13 @@
+<odoo>
+    <template id="add_logo_to_website" inherit_id="website.layout_logo_show" customize_show="False" name="Show Logo">
+        <xpath expr="//header//a[hasclass('navbar-brand')]" position="after">
+            <a href="/" class="navbar-brand logo">
+                <span t-field="website.logo" t-options="{'widget': 'image'}" role="img" t-attf-aria-label="Logo of {{website.name}}" t-att-title="website.name"/>
+            </a>
+        </xpath>
+        <xpath expr="//header//a[hasclass('navbar-brand')]" position="attributes">
+            <attribute name="class" add="d-none" separator=" "/>
+            <attribute name="style" add="Max-width:0px;margin:0px;"/>
```

Comment:
I missed this, you don't need 'ensure' that the element is 'hidden'... because it is not present in the layout (d-none).

Reply:
Yes, you can remove this.